jnbs/Digital_Crop_V3.ipynb

```diff
@@ -0,0 +1,4656 @@
+       "\n",
+       "   TransplantingIrrigationHours  TransplantingIrrigationSource  ...  \\\n",
+       "0                           5.0                              0  ...   \n",
+       "1                           5.0                              0  ...   \n",
+       "2                           4.0                              0  ...   \n",
+       "3                           2.0                              0  ...   \n",
+       "4                           9.0                              0  ...   \n",
+       "\n",
+       "   diff_days_Harv_Thresh  diff_days_Tillage_RcNursEst  \\\n",
+       "0                      0                        -23.0   \n",
+       "1                     29                        -28.0   \n",
+       "2                     30                        -10.0   \n",
+       "3                     27                          1.0   \n",
+       "4                      2                        -28.0   \n",
+       "\n",
+       "   diff_days_Tillage_Seeding  diff_days_Tillage_Thresh  \\\n",
+       "0                          1                       119   \n",
+       "1                          2                       159   \n",
+       "2                         44                       195   \n",
+       "3                         31                       196   \n",
+       "4                          1                       136   \n",
+       "\n",
+       "   diff_days_Tillage_Harv  diff_days_RcNursEst_Thresh  \\\n",
+       "0                     119                       142.0   \n",
+       "1                     130                       187.0   \n",
+       "2                     165                       205.0   \n",
+       "3                     169                       195.0   \n",
+       "4                     134                       164.0   \n",
+       "\n",
+       "   diff_days_RcNursEst_Harv  diff_days_RcNursEst_Seeding  \\\n",
+       "0                     142.0                         24.0   \n",
+       "1                     158.0                         30.0   \n",
+       "2                     175.0                         54.0   \n",
+       "3                     168.0                         30.0   \n",
+       "4                     162.0                         29.0   \n",
+       "\n",
+       "   diff_days_Seeding_Thresh  diff_days_Seeding_Harv  \n",
+       "0                       118                     118  \n",
+       "1                       157                     128  \n",
+       "2                       151                     121  \n",
+       "3                       165                     138  \n",
+       "4                       135                     133  \n",
+       "\n",
+       "[5 rows x 46 columns]"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Dropping unrequired Columns containing non-numerical values:\n",
```

Comment:
Is there any concrete reasons for your choice here?

Reply:
Why not one-hot encode the categorical columns? I didn't go through the data, I'm just asking to understand your thought process here

Reply:
Actually, those were some text columns and date columns, I had extracted the number of days and other required info from those..

Reply:
Cool, for the models you could also give Catboost, LightGBM, and XGBoost a try if you are going with Random Forest models

Reply:
> Cool, for the models you could also give Catboost, LightGBM, and XGBoost a try if you are going with Random Forest models

Sure, I shall try these in the upcoming versions